tests/api/test_bridge.py

```diff
@@ -0,0 +1,26 @@
+from pprint import pprint
+
+
+class TestSaiBridge:
+    # object with no parents
+
+    def test_bridge_create(self, npu):
+        commands = [
+            {
+                'name': 'bridge_1',
+                'op': 'create',
+                'type': 'SAI_OBJECT_TYPE_BRIDGE',
+                'attributes': ['SAI_BRIDGE_ATTR_TYPE', 'SAI_BRIDGE_TYPE_1Q'],
```

Comment:
As per SAI specification, only one .1Q bridge is supported. It's created implicitly as a default .1Q bridge. So, most likely this will fail on real implementation. I'd suggest to create .1D bridge instead.
Also, just for your reference: https://github.com/opencomputeproject/SAI-Challenger/blob/main/tests/ut/test_bridge_ut.py

Reply:
i have removed this script untill i figure out this issue